code/modules/antagonists/lost_crew/character/_character.dm

```diff
@@ -0,0 +1,33 @@
+/// Datum for controlling the base character, such as species, scarring, styles, augments etc
+/datum/corpse_character
+	/// Species type to spawn with
+	var/datum/species/species_type = /datum/species/human
+
+/datum/corpse_character/proc/apply_character(mob/living/carbon/human/fashionable_corpse, list/saved_objects)
+	fashionable_corpse.set_species(species_type)
+	fashionable_corpse.fully_replace_character_name(fashionable_corpse.real_name, fashionable_corpse.generate_random_mob_name())
+
+/// Not really all roundstart species, but plasmaman is a bit too flamboyant and felinids aren't interesting
+/datum/corpse_character/roundstart
+	var/list/possible_species = list(
+		/datum/species/human = 10,
+		/datum/species/lizard = 2,
+		/datum/species/ethereal = 1,
+		/datum/species/moth = 1,
+		)
```

Comment:
This doesn't account for config. If you actually wanted to be "roundstart species" you should base it off the roundstart species list.